ichnaea/db.py

```diff
@@ -83,6 +83,8 @@ def db_worker_session(database, commit=True):
     """
     try:
         session = database.session()
+        if commit:
+            session.begin_nested()
```

Comment:
This is the bit that will change task behaviour in production, for more than just station updating. If there is a general slowdown, I'll want to ensure ``begin_nested`` only runs in tests.